src/main/java/com/moodybluez/enterprise/dao/IDateDAO.java

```diff
@@ -0,0 +1,13 @@
+package com.moodybluez.enterprise.dao;
+
+import com.moodybluez.enterprise.dto.Date;
+
+public interface IDateDAO {
```

Comment:
Created DAO Interfaces to be used with @AutoWired (Polymorphism optimization)

Reply:
Looks very good.

Since interfaces are contracts, we usually JavaDoc them thoroughly.